pkg/pipelines/tekton/templates.go

```diff
@@ -88,8 +88,8 @@ const (
 )
 
 var (
-	FuncRepoRef       = "knative/func"
-	FuncRepoBranchRef = "main"
+	FuncRepoRef       = "openshift-knative/kn-plugin-func"
+	FuncRepoBranchRef = "serverless-1.30"
```

Comment:
shoudn't it be "serverless-1.31" ?

Reply:
🤦 thanks!

Reply:
I'll create another PR, sorry about that. Too close numbers probably.